### ISSUE TITLE
Debug через консоль в хроме.

### ISSUE THREAD

**Issue by geroddokar** [bem/bem-forum-content-ru#1019]
При вызове у button событие click() не происходит клик. Как вызвать через консоль?

**Comment by tadatuta** [comment]
@geroddokar было бы неплохо чуть более подробнее описать задачу.

Если я правильно угадываю, то речь о том, как стриггерить БЭМ-событие из консоли?
Если так, то:
```js
modules.require('jquery', function($) {
    $('.button').click(); // это вызов DOM-события
    $('.button').bem('button').click(); // это вызов БЭМ-события
})
```

Как видно из кода, получить экземпляр БЭМ-блока, имея на руках ссылку на jQuery-цепочку, можно с помощью метода `.bem('имя-блока')`.

**Comment by geroddokar** [comment]
Да, простите. Я пытаюсь автоматизировать действия на сайте. Столкнулся с тем что через скрипт я не могу кликнуть на кнопку. Скрипт я добавляю через браузер. Ругается на Uncaught ReferenceError: modules is not defined(…) Что нужно подключить? 
Кнопка:
<button class="button button__control i-bem button_js_inited" role="button" type="button" data-bem="{&quot;button&quot;:{}}"><span class="button__text">Отправить</span></button>

**Comment by tadatuta** [comment]
Тогда нужно еще больше подробностей. А еще лучше — репозиторий с кодом, где можно воспроизвести проблему.

**Comment by geroddokar** [comment]
Репозитория как такового нет. Так как пишу для себя, и скажем так это тестовый вариант. Клики на ссылки, дргуие элементы форм проходят нормально. А вот с кнопками проблема. Мои действия: 
1) Открыл сайт
2) Открыл консоль
3) ввел скрипт для теста
4) Смотрю на результат.
Сам скрипт прост. Часть которая не работает это клик по кнопке.  в изначальном варианте было так:
document.getElementsByClassName('classname')[0].click()
Но действий не следовало за ним.
после чего по вашему примеру я поменял на 
modules.require('jquery', function($) {
   document.getElementsByClassName('classname')[0].click(); // это вызов DOM-события
    document.getElementsByClassName('classname')[0].bem('button').click(); // это вызов БЭМ-события
})
Ругается или на "Uncaught ReferenceError: modules is not defined(…)" в хроме, 
в IE на "Не удалось получить свойство "bem" ссылки, значение которой не определено или является NULL"

**Comment by apsavin** [comment]
Какой сайт?
Тут что, клуб "Основы web scraping на bem.info"?)

**Comment by geroddokar** [comment]
Например Ссылка на сайт https://auto.ru/cars/all/?beaten=1&customs_state=1&advanced_search=1&show_sales=1&listing=listing
Кнопки перелистывание страниц.

**Comment by tadatuta** [comment]
Открыл страницу по этой ссылке, написал в консоль буквально следующее:
```js
modules.require('jquery', function($) {
    $('.pager__next').bem('button').emit('click');
});
```

Получил переход на следующую страницу в любом браузере.

**Comment by geroddokar** [comment]
Спасибо, Помогло. Простите что с опозданием.
